build.gradle

```diff
@@ -21,6 +21,9 @@ idea.project.settings {
 
 import net.ltgt.gradle.errorprone.CheckSeverity
 
+project('jib-gradle-plugin-extension-api') {
+  apply plugin: 'java-library'
+}
```

Comment:
Is there an elegant way to merge this with the Maven block below?

Reply:
you could do something like?
```
["jib-gradle-plugin-extension", "jib-maven-plugin-extension"].each { projectName ->
  project(projectName) {
    apply plugin: 'java-library'
  }
}
```
or some variation of this? I think you can call `apply` direction on the `Project` object? Nothing great I can think of.

Maybe in the subprojects section below you can check
```
if(project in [extension-api-projects list]) { 
  apply plugin: 'java-library' 
}
```

Reply:
The `if` check in the `subprojects` sections didn't work. I can go with the first suggestion.

Reply:
Hrmm... this is what worked for me in the end:
```
if ([rootProject.project('jib-maven-plugin-extension-api')].contains(project)) {
  apply plugin: 'java-library'
}
```
but in the end, I mean they're all kinda gross :\